byzcoin/bcadmin/main.go

```diff
@@ -295,9 +434,458 @@ func add(c *cli.Context) error {
 	return nil
 }
 
-func keys(c *cli.Context) error {
-	s := darc.NewSignerEd25519(nil, nil)
-	fmt.Println("Identity:", s.Identity())
-	fmt.Printf("export PRIVATE_KEY=%v\n", s.Ed25519.Secret)
+func key(c *cli.Context) error {
+	newSigner := darc.NewSignerEd25519(nil, nil)
+	lib.SaveKey(newSigner)
+
+	save := c.String("save")
```

Comment:
Something is wrong with the formatting here. Look into configuring gofmt-on-save for your editor.